### PR TITLE
Created sending email for session accept/reject

### LINKED ISSUE #4209: Email for session submission, approval and rejection
Parent Issue #4196

Emails for following are to be done in this issue:

- [x] Email for session submission to Event Organizer
- [x] Email for session acceptance/rejection to Speaker
- [x] Email for session acceptance/rejection to Organizer

### PR DESCRIPTION
**Pull request by magdalenesuo**
Fixes #4209

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `nextgen` branch.
- [X] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [X] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Created sending email for session accept/reject

#### Changes proposed in this pull request:

- [X] Email for session acceptance/rejection to Speaker
- [X] Email for session acceptance/rejection to Organizer

### CODE REVIEW

File: app/api/sessions.py [hunk 1]
```diff
@@ -194,11 +197,32 @@ def before_get_object(self, view_kwargs):
             event = safe_query(self, Event, 'identifier', view_kwargs['event_identifier'], 'identifier')
             view_kwargs['event_id'] = event.id
 
+    def after_update_object(self, session, data, view_kwargs):
+        """ Send email if session accepted or rejected """
+        if session.state == 'accepted' or session.state == 'rejected':
```
Comment (shubham-padia) [inline app/api/sessions.py]: Not sure, but wouldn't this send mail every time the resource is updated and the session is either accepted or rejected ?

File: app/api/sessions.py [hunk 2]
```diff
@@ -194,11 +197,32 @@ def before_get_object(self, view_kwargs):
...
+            # Email for speaker
+            speakers = session.speakers
+            for speaker in speakers:
+                frontend_url = get_settings()['frontend_url']
+                link = "{}/{}/sessions/{}" \
```
Comment (SaptakS) [inline app/api/sessions.py]: I think the link will give wrong value. It should be `"{}/events/{}/sessions/{}"`

File: app/api/sessions.py (hunk 1 above)
Comment (magdalenesuo) [inline app/api/sessions.py]: @shubham-padia Yup! updated (:

File: app/api/sessions.py (hunk 2 above)
Comment (magdalenesuo) [inline app/api/sessions.py]: @SaptakS Updated :sweat_smile: please review again

**Pull request merged**